R/derive_param_maxtemp.R

```diff
@@ -0,0 +1,136 @@
+#' Deriving maximum temperature record
+#'
+#' @description
+#' Create a summary record for maximum temperature per subject per period.
+#'
+#' @param dataset Input data set
+#'
+#'  Input data set should have the temperature record which will be derived by
+#'  using the function `derive_param_fever_occur`.
+#'
+#' @param filter_faobj filter for fever record
+#'
+#' *Default: "FEVER"*
+#' *Permitted Value: A Character Scalar*
+#'
+#' Helps to filter the fever record by passing the value from `FATESTCD` which
+#' indicates the fever records.
+#'
+#' @param by_vars Grouping variables
+#'
+#' *Default: vars(USUBJID,FAOBJ,ATPTREF)*
+#' *Permitted Value: A Character vector*
+#'
+#'  Pass the `by_vars` in `vars()` based on the by_vars variables the maximum
+#'  temperature will be derived from `VSSTRESN`.
+#'
+#' @param test_maxtemp Value for `FATEST`
+#'
+#' *Default*: "Maximum temperature"
+#' *Permitted Value: A Character scalar*
+#'
+#' Pass a value for `FATEST` which will be indicate the maximum temperature
+#' records
+#'
+#' @param testcd_maxtemp
+#'
+#' *Default*: "MAXTEMP"
+#' *Permitted Value: A Character scalar*
+#'
+#' Pass a value for `FATESTCD` which will be indicate the maximum temperature
+#' records. refer the default value.
+#'
+#' @return
+#' The input data set with new records `FATESTCD = MAXTEMP`indicating the
+#' maximum temperature records based on the specified variables in `by_vars`.
+#' `DTYPE` will be populated as `MAXIMUM`. `FATEST`will be populated as
+#' specified in `test_maxtemp`and `FATESCD` will be populated as specified in
+#' `testcd_maxtemp`. Maximum temperature values will be populated in `AVAL`.
+#'
+#' @export
+#'
```

Comment:
You can use the `der_adxx` for `@family` and `@keywords`, but this is needed to build the pkgdown site.